server/mail/templates/index.js

```diff
@@ -176,7 +177,8 @@ const replaceHTMLContent = (type, html = '', email, data, language, domain) => {
 		}
 
 		html = html.replace(/\$\{name\}/g, email || ''); //
-		html = html.replace(/\$\{currency\}/g, data.fee_coin || data.currency.toUpperCase()); //
+		let currency = data.fee_coin || data.currency || '';
```

Comment:
@hungvuit94 why `let currency = data.fee_coin || data.currency || '';` ? currency should be data.currency I suppose.